### PR TITLE
Set up Frontend CI with Azure Pipelines

### PR DESCRIPTION
**Pull request by EhartGMU**

### CODE REVIEW

**Review by MMooKow — APPROVED** [review]
Looks good

File: frontend-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#DOCKER TO BE ADDED LATER
+#Front End
+trigger:
+- main
```
Comment (escalonn) [inline frontend-pipelines.yml:4]: this branch does not exist

File: frontend-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- stage: NG
+  #Installs/Builds angular app
+  jobs:
+    - job: NGbuild
+      displayName: NG install/build
+      steps:
+        - script: |
+            npm install -g @angular/cli
+            npm ci
+            ng build --prod
```
Comment (escalonn) [inline frontend-pipelines.yml:37]: there's a few issues with this step

**Comment by escalonn** [comment]
also, the tests should be run

**Comment by escalonn** [comment]
the same issues exist with https://github.com/210726-wvu-net-ext/Group2-P3-Pet_Insurance/pull/9

**Pull request merged**